### ISSUE TITLE
Google Drive Synchronisation?

### ISSUE THREAD

**Issue by Integralist** [android-password-store/Android-Password-Store#235]
Is it possible to put your password store into Google Drive (or whatever cloud storage) and then sync from there to this app?

I appreciate doing this reduces the security down to the strength of your gpg passphrase alone. 

But if you feel confident you have a strong and varied phrase that's computationally intensive then this could be a simpler alternative, I guess?

**Comment by Lindenk** [comment]
Funnily enough, I have the opposite problem. I just bought a new phone and apparently my passwords were synced with Google drive along with all connection settings to my repo. Syncing settings such as autocompletion are convenient, but I'd prefer to not have the store backed up by Google.

 
It looks like this is doable which the [android api](https://developer.android.com/guide/topics/data/autobackup). Google must have switched to backing up everything by default at some point.

**Comment by grensjo** [comment]
@Lindenk Yes, this happened to me as well, and goes completely against the behavior I would expect from the app.

One of the drawbacks of pass is that while the passwords themselves are encrypted, the folder and file names are not. Thus, for most use cases, sites and account names are stored in clear text. Despite this information being potentially sensitive I decided to accept this drawback, since I had made sure to only store the repo on devices that I control. But now it turns out that the repo is silently and invisibly auto-backed up to Google Drive, and all of that information has been uploaded to Google without my consent.

To be fair, I had explicitly opted in to app backups in my device settings, but on the other hand it did seem like a reasonable assumption that apps dealing with sensitive information would not upload it to Google without asking.

In conclusion: @zeapo, please disable auto-backups for this app. It is a simple [manifest change](https://developer.android.com/guide/topics/data/autobackup#EnablingAutoBackup), the affected line is [here](https://github.com/zeapo/Android-Password-Store/blob/master/app/src/main/AndroidManifest.xml#L11). As far as I can tell users cannot opt out on a per-app basis, so the only current workaround is to disable app backups altogether.

**Comment by zidhuss** [comment]
@Lindenk, @grensjo thanks for pointing this out. Sorry it took a while to get to.

I've made the change in 9496ed31fdd6525a738eb781e567a8a723d577c2. It will be in the next release!

**Comment by msfjarvis** [comment]
Since opting into Android's Auto Backup feature can't be done at runtime and most users including me absolutely do not want it, I'm going to close this issue. Synchronizing with Git is hard enough so I am not willing to manually sync with Google Drive as well.
